src/pretix/base/models/orders.py

```diff
@@ -1137,12 +1137,19 @@ def resend_link(self, user=None, auth=None):
                 attach_tickets=True,
             )
 
+    @property
+    def positions_with_tickets_ignoring_plugins(self):
+        return (op for op in self.positions.select_related('item') if op.generate_ticket)
+
     @property
     def positions_with_tickets(self):
-        for op in self.positions.select_related('item'):
-            if not op.generate_ticket:
-                continue
-            yield op
+        signal_response = allow_ticket_download.send(self.event, order=self)
```

Comment:
I know this is nit-picking, but semantically IMHO positions_with_tickets and positions, whose tickets can be downloaded are different. So I am not sure we should reuse the property-name, but instead add another property e.g. `positions_with_downloadable_tickets`?

Reply:
In theory, yes. I had it named differently in the meantime (something along the lines of "positions where download is allowed by plugins"). However, as this property is [documented](https://docs.pretix.eu/en/latest/development/api/ticketoutput.html#pretix.base.ticketoutput.pretix.base.ticketoutput.BaseTicketOutput.BaseTicketOutput.generate_order) as the property to use when generating tickets for download, I think we should keep the name.